### DISCUSSION TITLE
IllegalStack hook

### DISCUSSION THREAD

**Discussion by RVSkeLe** [BG-Software-LLC/WildTools#189] [Pending Feature Requests]
Hello, illegalstack has the possibility to set max Nether height, it would be nice to have the hook so that players cannot go above set nether height.
`
Nether:
    KillPlayersBelowNether: false
    BlockBuildingAboveNether: true
    BlockPlayersAboveNether: false
    NetherYLevel: 128
    ExcludeNetherWorldFromHeightCheck: []
    EnsureSafeTeleportLocationIfAboveCeiling: false
`

**Comment by OmerBenGera** [comment]
Add "IllegalStack" to the claiming-plugins list.
It doesnt fix the issue?

**Comment by RVSkeLe** [comment]
> Add "IllegalStack" to the claiming-plugins list. It doesnt fix the issue?

It does not unfortunately
